customersupport/templates/template.html

```diff
@@ -41,9 +41,9 @@
         </div>
         <div id="navbar" class="collapse navbar-collapse">
             <ul class="nav navbar-nav">
-                <li class="active"><a href="{{ url_for("new_call") }}">New Call</a></li>
+                <li><a href="{{ url_for("new_call") }}">New Call</a></li>
```

Comment:
Meh. You can skip it for now (unless @Daniel-J-Fisher feels strongly about it). It's a little template change later on if/when we want to add it.